util/errors/errors.go

```diff
@@ -23,12 +23,14 @@ func IgnoreContainerNotFoundErr(err error) error {
 	return err
 }
 
+// As close to side-effect free of a check of the transient or non-transient state of an error.
```

Comment:
Could you format this and the other comments in here as a [godoc](https://tip.golang.org/doc/comment) compabile comment please?

Reply:
I think I've complied with the godoc style in the new commit, but let me know. 🙌